acceptance/steps/privacy-policy.test.js

```diff
@@ -0,0 +1,41 @@
+const JestCucumber = require('jest-cucumber')
+const WebDriver = require('selenium-webdriver');
+const firefox = require('selenium-webdriver/firefox');
+
+const screen = {
+  width: 1024,
+  height: 768
+};
+
+const feature = JestCucumber.loadFeature('features/privacy-policy.feature');
+
+JestCucumber.defineFeature(feature, test => {
+  test('Privacy policy page contains expected heading', ({ given, when, then }) => {
+
+    let url = "";
+    let driver;
+
+    given('I am a Cyber Capable Person', () => {
+      driver = new WebDriver.Builder()
```

Comment:
Perhaps consider this duplication as well if we can?

Reply:
Sorted in 1931a68